client/src/components/Modal/__tests__/NumberInputModal.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import NumberInputModal from '../NumberInputModal';
+
+const noop = () => {};
+
+test('renders correctly', () => {
+  const tree1 = renderer.create(
+    <NumberInputModal
+      title="test"
```

Comment:
if you put different text in the attributes then we can more easily detect someone mixing prop names up. ie if you set both `title="test"` and `placeHolder="test"` and someone puts `{placeHolder}` where there is supposed to be `{title}` then the snapshot test wouldn't catch that bug. If the attributes are all different there would be a change that the snapshot test would catch. As such I recommend doing what I did for the others - for string props put the name of the prop as the test value.